src/System.Reflection.Metadata/tests/TestUtilities/LoaderUtilities.cs

```diff
@@ -33,7 +34,18 @@ public static void LoadPEAndValidate(byte[] peImage, Action<PEReader> validator,
                 }
             }
 
-            File.Delete(tempFile);
+            for (int i = 0; i < 10; i++)
```

Comment:
Let me merge in my change now to avoid further failed test runs and file an issue to follow up on why the file stays locked.